src/components/LinkManager/LinkList/LinkOverview/LinkOverView.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+import styles from "./LinkOverview.module.scss";
+import { Link as LinkType } from "@/src/constants/types";
+import getElapsedTime from "@/src/utils/getElapsedTime";
+import getFormattedDate from "@/src/utils/getFormattedDate";
+import { LinkImageEmptyCase } from "@/src/asset/";
+import Image from "next/image";
+
+interface LinkOverview {
+  link: LinkType;
+}
+
+export default function LinkOverview({ link }: LinkOverview) {
+  const { imageSource: imageUrl, createdAt, description, title, url } = link;
+  const createDate = getElapsedTime(createdAt);
+  const formatDate = getFormattedDate(createdAt);
+
+  // 서버 Image-url이 문제가 있어서 그냥 null처리 했읍니다
+  function fixedUrl(url: string) {
+    if (!url) return null;
```

Comment:
한줄이라도 중괄호를 무조건 작성하고, 조건식을 직관적으로 써주는게 좋습니다!
```suggestion
    if (url === '') {
     return null;
    }
```